plugins/rapture/nexus-coreui-plugin/src/main/resources/static/rapture/NX/coreui/controller/RepositoriesCMA.js

```diff
@@ -51,14 +51,6 @@ Ext.define('NX.coreui.controller.RepositoriesCMA', {
     'repository-default': {
       file: 'database_green.png',
       variants: ['x16', 'x32']
-    },
-    'repository-online': {
-      file: 'tick.png',
-      variants: ['x16']
-    },
-    'repository-offline': {
-      file: 'cross.png',
-      variants: ['x16']
```

Comment:
Initially I made a field for online/offline and showed an image. But I removed that in favour of a status column like in NX2 that include both the online/offline and the remote status. This way for non proxy repositories it will not look strange to have empty columns.